File: justaminute.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main()
{
    int t, mins, secs;
    cin >> t;
    for (int i = 0; i < t; i++)
    {
        int minin, secin;
        cin >> minin >> secin;
        mins += minin;
        secs += secin;
    }
    double avg = (double)secs/(double)(mins*60);
    if (avg <= 1)
        cout << "measurement error" << endl;
    else
        cout << setprecision(10) << fixed << avg << endl;
}

File: justaminute.py
mins, secs = 0, 0
for i in range(int(input())):
    min, sec =  [int(j) for j in input().split()]
    mins += min
    secs += sec
avg = float(secs)/float(mins*60)
if avg <= 1:
    print("measurement error")
else:
    print("%.10f" % avg)
